scripts/release_scripts/run_macaron.sh

```diff
@@ -0,0 +1,394 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2023 - 2023, Oracle and/or its affiliates. All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/.
+
+# This script runs the Macaron Docker image.
+
+if [[ -z ${MACARON_IMAGE_TAG} ]]; then
+    MACARON_IMAGE_TAG="latest"
+fi
+
+IMAGE="ghcr.io/oracle-samples/macaron"
+
+# Workspace directory inside of the container.
+MACARON_WORKSPACE="/home/macaron"
+
+# The entrypoint to run Macaron or the Policy Engine.
+# It it set by default to macaron.
+# We use an array here to preserve the arguments as provided by the user.
+entrypoint=()
+
+# The action to run for each entrypoint.
+# For example: `macaron analyze` or `macaron dump_defaults`
+action=()
+
+# Arguments whose values changed by this script before being passed to the Macaron Docker image.
+# These are arguments for macaron entrypoint.
+#   -dp/--defaults-path DEFAULTS_PATH: The path to the defaults configuration file.
+#   -h/--help:  Show the help message and exit.
+#   -lr/--local-repos-path LOCAL_REPOS_PATH: The directory where Macaron looks for already cloned repositories.
+#   -po/--policy POLICY: The path to a policy file.
+#   -t/--personal_access_token PERSONAL_ACCESS_TOKEN: The GitHub personal access token, which is mandatory for running analysis.
+#   -v/--verbose: Run Macaron with more debug logs.
+# These are arguments for policy_engine entrypoint.
+#   -f/--file FILE: Replace policy file.
+#   -d/--database DATABASE: Database path.
+argv_main=()
+
+# Arguments whose values changed by this script before being passed to the Macaron Docker image.
+# These are arguments will be specific to action in macaron entrypoint.
```

Comment:
```suggestion
# These are the sub-commands for a specific action.
```

Does that make sense?

Reply:
Yep, it sounds good to me.

Reply:
Fixed in https://github.com/oracle-samples/macaron/pull/76/commits/8694a7fe8b5617a73598be3d26945989ee457684